### PR TITLE
PAF-25: create initial unit tests

### PR DESCRIPTION
**Pull request by Rhodine-orleans-lindsay**
## What
- Create initial unit tests

## Why
- To allow for testing in pipeline

### CODE REVIEW

File: .nycrc [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "reporter": [
+    "lcov",
+    "text-summary"
+  ],
+  "check-coverage": true,
+  "statements": 82,
+  "branches": 68,
+  "functions": 83,
+  "lines": 82,
```
Comment (sulthan-ahmed) [inline .nycrc:10]: is this going to fail if we don't hit these numbers?

File: package.json [hunk 2]
```diff
@@ -15,8 +15,9 @@
     "start": "node server.js",
     "start:dev": "hof-build watch",
     "dev": "hof-build watch --env",
-    "test": "yarn run test:lint",
+    "test": "yarn run test:lint && yarn run test:unit",
     "test:lint": "eslint .",
+    "test:unit": "NODE_ENV=test nyc _mocha \"test/_unit/**/*.spec.js\"",
```
Comment (sulthan-ahmed) [inline package.json:20]: I would prefer to also just have a unit test script on it's own but not important for now.

File: .nycrc (hunk 1 above)
Comment (Rhodine-orleans-lindsay) [reply .nycrc:10]: yes it would but they can be changed if needs be

File: package.json (hunk 2 above)
Comment (Rhodine-orleans-lindsay) [reply package.json:20]: not sure what you mean by unit test script on it's own?

Comment (sulthan-ahmed) [reply package.json:20]: This I believe runs the unit tests and also the coverage.  When I'm fixing unit tests, I don't really want the coverage malarky in my way so I just want a script to run the unit tests 😄

File: .nycrc (hunk 1 above)
Comment (sulthan-ahmed) [reply .nycrc:10]: Do you think having limits is going to block us?

File: package.json (hunk 2 above)
Comment (sulthan-ahmed) [reply package.json:20]: don't worry, I'll add it myself

File: .nycrc (hunk 1 above)
Comment (Rhodine-orleans-lindsay) [reply .nycrc:10]: I can remove this for now and add it in once we are closer to finishing everything?

Comment (sulthan-ahmed) [reply .nycrc:10]: Sounds good.  Do you think when we do add it, we need to discuss and agree it as a whole in terms of what our standards are in HOF and the wider HO?

Comment (Rhodine-orleans-lindsay) [reply .nycrc:10]: yeah, I think that would be a good idea - encourage more consistency across the services 👍🏾

Comment (Rhodine-orleans-lindsay) [reply .nycrc]: I have removed this file now

**Review by sulthan-ahmed — APPROVED** [review]
lgtm, it's just the pipeline is failing for some reason.

nice work

**Comment by Rhodine-orleans-lindsay** [comment]
> lgtm, it's just the pipeline is failing for some reason.
> 
> nice work

yes, the yarn.lock needed to be updated which I have done now.

**Pull request merged**